studio/src/fields/multifield.js

```diff
@@ -96,6 +96,14 @@ class MasMultifield extends LitElement {
     /* c8 ignore start */
     // Handle drag start
     dragStart(e, index) {
+        const activeElement = this.shadowRoot.activeElement;
+        if (
+            activeElement &&
```

Comment:
debugged it a bit and activeElement seems to be null during normal drag&drop 
I am not sure if activeElement is ever 'mas-multifield', maybe we could just have:
```
if (activeElement) {
    e.preventDefault();
    return;
}
```

or if it ever happens that an active element is mas-multifield, then:
```
if (activeElement?.tagName.toLowerCase() !== 'mas-multifield') {
    e.preventDefault();
    return;
}
```
But i'd try option 1 and see if it works well - might be enough